Clinical-Genomics/trailblazer
# Fixes more obsolete dependencies

This PR upgrades trailblazers nuxt dependencies

How to test:
1. install on clinical-db stage, both ui and back-end 
1. dbcopyprodtostage trailblazer
1. us
1. login to trailblazer

Expected outcome:
all production analyses are visible also in stage (because of the dbcopy)
